src/dbnode/storage/limits/query_limits.go

```diff
@@ -209,22 +255,29 @@ func (q *lookbackLimit) checkLimit(recent int64) error {
 }
 
 func (q *lookbackLimit) start() {
-	ticker := time.NewTicker(q.options.Lookback)
+	q.ticker = time.NewTicker(q.options.Lookback)
```

Comment:
any reason why ticker is on the object now? doesn't look used outside of this method

Reply:
Will remove. Had originally been planning to call `ticker.Reset` when updating but chose against it because it seemed cleaner to just replace the ticker rather than mutate